src/Core/StaticConfigTrait.php

```diff
@@ -68,12 +68,12 @@ trait StaticConfigTrait
      * ```
      *
      * @param array<string, mixed>|string $key The name of the configuration, or an array of multiple configs.
-     * @param object|array<string, mixed>|null $config An array of name => configuration data for adapter.
+     * @param mixed $config The value for the config key. Generally an array of name => configuration data for adapter.
```

Comment:
The incorrect docblock in 4.x is how we ended up with incorrect types in 5.x, so it should be fixed there too.